src/webrtc/groupCall.ts

```diff
@@ -517,6 +532,28 @@ export class GroupCall extends TypedEventEmitter<
         return true;
     }
 
+    public async setVadMicrophoneMuted(muted: boolean): Promise<boolean> {
+        // hasAudioDevice can block indefinitely if the window has lost focus
+        if (!(await this.client.getMediaHandler().hasAudioDevice())) {
```

Comment:
Even if not necessarily necessary (_words_), I think it would be good to have the `muted` check here, just to be sure